integration_tests/tests/mod.rs

```diff
@@ -39,8 +42,8 @@ pub enum Operation {
     MutualClose,
     CustomerClose,
     MerchantExpiry,
-    Store,
-    Restore,
+    Store(MaliciousParty, &'static str),
+    Restore(MaliciousParty, &'static str),
```

Comment:
note (optional): I think this example doesn't require it, but if you can feel free to expand the tuple enum variant into a C-style variant with names if it seems easier to understand e.g.
```
Store { party: MaliciousParty, tag: &'static str),
```